google/cloud/bigtable/admin/integration_tests/table_admin_integration_test.cc

```diff
@@ -325,39 +329,14 @@ TEST_F(TableAdminIntegrationTest, WaitForConsistencyCheck) {
   auto consistency_token =
       std::move(*consistency_token_resp->mutable_consistency_token());
 
-  // A retry loop that checks if the data replication has completed.
-  auto wait_for_consistency = [&]() -> Status {
-    auto retry = google::cloud::internal::LimitedTimeRetryPolicy<
-        bigtable::internal::SafeGrpcRetry>(std::chrono::minutes(2));
-    auto backoff = google::cloud::internal::ExponentialBackoffPolicy(
-        std::chrono::milliseconds(5), std::chrono::minutes(2), 2.0);
-
-    Status status;
-    do {
-      auto result = client_.CheckConsistency(table_name, consistency_token);
-      if (result.ok()) {
-        // Consistency has been achieved. Break the loop.
-        if (result->consistent()) return Status();
-        status = Status(StatusCode::kUnavailable, "Not consistent yet");
-      } else {
-        status = std::move(result).status();
-      }
-      if (!retry.OnFailure(status)) {
-        // The retry policy is exhausted or the error is not retryable.
-        return status;
-      }
-      std::this_thread::sleep_for(backoff.OnCompletion());
-    } while (!retry.IsExhausted());
-    return status;
-  };
-
-  // Verify that our clusters are eventually consistent.
-  auto result = wait_for_consistency();
-  ASSERT_STATUS_OK(result);
+  // Verify that our clusters are eventually consistent. This calls
```

Comment:
nit: we should not verifying anything about the clusters, that is the Bigtable service team job.  We are verifying that our functions complete under the assumption that Bigtable works.